sdk/include/opentelemetry/sdk/metrics/view/view.h

```diff
@@ -24,12 +26,15 @@ class View
   View(const std::string &name,
        const std::string &description   = "",
        AggregationType aggregation_type = AggregationType::kDefault,
+       nostd::shared_ptr<AggregationConfig> aggregation_config =
```

Comment:
nit  - we can use `std::shared_ptr` in sdk surface.

Reply:
done